kitsune/questions/models.py

```diff
@@ -392,6 +392,14 @@ def is_solved(self):
     def is_offtopic(self):
         return config.OFFTOPIC_TAG_NAME in [t.name for t in self.my_tags]
 
+    @property
```

Comment:
Nit (since it's only used once so far). Maybe use `@cached_property` (from `functools`) instead, since this performs a DB query?

Reply:
great idea!